test/testdrive/pg_get_constraintdef.td

```diff
@@ -0,0 +1,24 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Without EXTENDED, SHOW TYPES shouldn't have anything to return.
+
+! SELECT pg_get_constraintdef(885)
```

Comment:
I'm not convinced this needs to be in both td and slt. I think just the slt tests are sufficient.